csrankings-a.csv

```diff
@@ -1478,6 +1478,7 @@ Andrew Colin Rice,University of Cambridge,https://www.cl.cam.ac.uk/~acr31,bxvOPw
 Andrew Crabtree,University of Nottingham,https://www.nottingham.ac.uk/computerscience/people/andy.crabtree,LX_P5GYAAAAJ
 Andrew D. Miller 0001,IUPUI,https://soic.iupui.edu/people/andrew-miller,U5QeuUcAAAAJ
 Andrew D. Calway,University of Bristol,http://people.cs.bris.ac.uk/~andrew/,jtlbdOoAAAAJ
+Andrew D. Gordon,University of Edinburgh,https://www.inf.ed.ac.uk/people/staff/Andrew_Gordon.html,mfBjUiIAAAAJ

```

Comment:
I don't think Andy is full-time at Edinburgh, given his MS appointment. Please clarify.

Reply:
I removed Andrew D. Gordon, since he is not full-time in Edinburgh.